src/data/systemIntake.ts

```diff
@@ -84,6 +84,7 @@ export const prepareSystemIntakeForApi = (systemIntake: SystemIntakeForm) => {
       id: systemIntake.id
     }),
     status: systemIntake.status,
+    requestType: 'NEW',
```

Comment:
This is hardcoded as `NEW` for now since the intake only supports new intakes right now. In my next PR, I'll hook up the frontend and backend work, which will involve updating our static types as well as passing the selected request type.